js/src/admin/components/PermissionGrid.js

```diff
@@ -366,4 +391,20 @@ export default class PermissionGrid extends Component {
   scopeControlItems() {
     return new ItemList();
   }
+
+  getExtensionPermissions(type) {
```

Comment:
Again, would be nice for this to be part of some ExtensionManager util or something, so it's not duplicated across this and ExtensionPermissionGrid, and so that the PermissionGrids don't need to concern themselves with the implementation of how extensions register / store permissions